### PR TITLE
  #9935 changed the link from /guides/configuration-management/creating-a-meshrey-design to /guides/creating-a-meshrey-design

### PR DESCRIPTION
**Pull request by muthu9594**
**Notes for Reviewers**

This PR fixes #

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 7743b9e24aa77c367c52a89d0c643badadc0ae00 at: https://65a638b878ba5f8b02cedfea--meshery-docs-preview.netlify.app

File: docs/pages/guides/configuration-management/creating-a-meshery-design.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 layout: enhanced
 title: Creating a Meshery Design
 abstract: Learn how to create a Meshery design from scratch or from a template.
-permalink: guides/configuration-management/creating-a-meshery-design
+permalink: guides/creating-a-meshery-design
```
Comment (vishalvivekm) [inline docs/pages/guides/configuration-management/creating-a-meshery-design.md:5]:
```suggestion

```

permalink is not to be changed @muthu9594

**Comment by vishalvivekm** [comment]
@muthu9594 Please follow community code of conduct i.e. don't raise PRs without getting the issue assigned to you.

 I understand that you may not have had the chance to go through the [contributing guidelines](https://layer5.io/community/handbook/contribution#:~:text=Contributor%20Guidance%3A) and code of conduct outlined in the [community handbook](https://layer5.io/community/handbook). 
Please go through these documents and I hope you'll keep them in mind for future.

#9935

**Comment by saurabh100ni** [comment]
Hi @muthu9594  ,
Thanks for your PR! I encourage you to share this in the Website Meeting tomorrow at 6:30 PM IST. Share your screen, and walk us through it. It's cool to take and share feedback. Check the [Doc](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit) here to include this PR.

You're welcome at the meeting

Check to [community calendar](https://layer5.io/community/calendar) for more details of meetings at Layers.

**Review by leecalcote — DISMISSED** [review]
Accepted @vishalvivekm's code suggestion.

**Comment by l5io** [comment]
🚀 Preview for commit f3543086f97eb78d56d42c88f108b439b1ae3812 at: https://65af24c0d63246862e7c2d87--meshery-docs-preview.netlify.app

File: docs/pages/guides/configuration-management/creating-a-meshery-design.md [hunk 2]
```diff
@@ -2,7 +2,6 @@
 layout: enhanced
 title: Creating a Meshery Design
 abstract: Learn how to create a Meshery design from scratch or from a template.
```
Comment (leecalcote) [inline docs/pages/guides/configuration-management/creating-a-meshery-design.md:4]:
```suggestion
abstract: Learn how to create a Meshery design from scratch or from a template.
permalink: guides/creating-a-meshery-design
```

File: docs/_data/toc.yml [hunk 3]
```diff
@@ -140,7 +140,7 @@
       url: guides/configuration-management
       children:
       - title: Creating a Meshery Design
-        url: guides/configuration-management/creating-a-meshery-design
+        url: guides/creating-a-meshery-design
```
Comment (leecalcote) [inline docs/_data/toc.yml:143]:
```suggestion
        url: guides/configuration-management/creating-a-meshery-design
```

File: docs/pages/guides/configuration-management/creating-a-meshery-design.md (hunk 1 above)
Comment (leecalcote) [inline docs/pages/guides/configuration-management/creating-a-meshery-design.md:5]:
```suggestion
permalink: guides/configuration-management/creating-a-meshery-design
```

**Review by leecalcote — APPROVED** [review]

**Pull request merged**